src/services/admission/admission-formats.ts

```diff
@@ -0,0 +1,28 @@
+export interface DecisionInfoEntry {
+    _id?: string;
+    userId: string;
+    name: string;
+    status: DecisionStatus;
+}
+export interface UpdateEntries {
+    entries: DecisionInfoEntry[];
+}
+export interface DecisionInformationEntry {
+    userId: string;
+    status: DecisionStatus;
+    response: DecisionResponse;
+    reviewer: string;
+    emailSent: boolean;
+}
```

Comment:
is this not the exact same thing as the class from `decision-db.ts`? se that instead here